Source/Scene/Scene.js

```diff
@@ -955,6 +997,97 @@ define([
         return undefined;
     }
 
+    // These values were found by sampling the density at certain views and finding at what point culled tiles impacted the view at the horizon.
+    var heightsTable = [659.246, 1275.6501, 2151.1192, 3141.7763, 4777.5198, 6281.2493, 12364.307, 15900.765, 49889.0549, 78026.8259, 99260.7344, 120036.3873, 151011.0158, 156091.1953, 203849.3112, 274866.9803, 319916.3149, 493552.0528, 628733.5874, 1027505.709];
+    var densityTable = [2.0e-4, 1.0e-4, 7.0e-5, 5.0e-5, 4.0e-5, 3.0e-5, 1.9e-5, 1.0e-5, 8.5e-6, 6.2e-6, 5.8e-6, 5.3e-6, 5.2e-6, 5.1e-6, 4.2e-6, 4.0e-6, 3.4e-6, 2.6e-6, 2.2e-6, 1.6e-6];
+
+    // Scale densities by 1e6 to bring lowest value to ~1. Prevents divide by zero.
+    for (var i = 0; i < densityTable.length; ++i) {
+        densityTable[i] *= 1.0e6;
+    }
+    // Change range to [0, 1].
+    var tableStartDensity = densityTable[0];
+    var tableEndDensity = densityTable[densityTable.length - 1];
+    for (var j = 0; j < densityTable.length; ++j) {
+        densityTable[j] = (densityTable[j] - tableEndDensity) / (tableStartDensity - tableEndDensity);
+    }
+
+    var tableLastIndex = 0;
+
+    function findInterval(height) {
```

Comment:
This is really similar to some spline code you wrote a few years ago.  It's not reasonable/possible to reuse that?